apps/src/templates/manageStudents/ManageStudentsTable.jsx

```diff
@@ -340,13 +341,15 @@ class ManageStudentsTable extends Component {
 
   familyNameFormatter(familyName, {rowData}) {
     const editedValue = rowData.isEditing ? rowData.editingData.familyName : '';
+    const inputDisabled = this.isTeacher(rowData.userType);
     return (
       <ManageStudentsFamilyNameCell
         id={rowData.id}
         familyName={familyName}
         isEditing={rowData.isEditing}
         editedValue={editedValue}
         sectionId={rowData.sectionId}
+        inputDisabled={inputDisabled}
```

Comment:
I'd lean toward calling this property `isTeacher`. That would better break up the components here. As is, I could foresee a future developer who wanted to disable the cell for a different reason potentially thinking all they have to do is change the code above to `const inputDisabled = this.isTeacher(rowData.userType) || otherReasonToDisableCell;`, without realizing that the `inputDisabled` property triggers a very specific tooltip that would not apply to that other reason.